BoxContentSDK/BoxContentSDK/Requests/BOXSearchRequest.h

```diff
@@ -22,6 +22,8 @@
 @property (nonatomic, readwrite, strong) NSArray *ancestorFolderIDs;
 @property (nonatomic, readwrite, strong) NSArray *contentTypes;
 @property (nonatomic, readwrite, strong) NSString *type;
+@property (nonatomic, readwrite, strong) NSString *metadataTemplateKey;
+@property (nonatomic, readwrite, strong) NSString *metadataScope;
```

Comment:
All the `NSString` and `NSArray` properties should be `copy`.

Nit: Consider giving the arrays explicit element types while we're changing this.